packages/core/src/contrib/Constraints.ts

```diff
@@ -116,7 +116,7 @@ export const objDict = {
     [t2, text1]: [string, any],
     [t3, text2]: [string, any]
   ): VarAD => {
-    const spacing = varOf(1.1); // arbitrary
+    const spacing = varOf(2.0); // arbitrary
```

Comment:
Might this cause regressions/changes in other examples?

Reply:
Umm this should really be an argument of the function. Anyway, this function is not used in this domain. Reverting this change.